draft-bmw-tls-pake13.md

```diff
@@ -388,6 +388,25 @@ Applications for which this leak is a problem can use the TLS Encrypted ClientHe
 
 # Security Considerations {#security}
 
+## Dictionary attack mitigation
+
+Because PAKE security is based on knowledge of a low-entropy secret,
+an attacker can perform a "dictionary attack" by repeatedly attempting to
+guess the low-entropy secret.
+
+Clients and servers should apply mitigations against dictionary attacks.
+Reasonable mitigations include rate-limiting authentication attempts,
+imposing a backoff time between attempts, or limiting the total number
+of attempts.
```

Comment:
Perhaps:

```suggestion
imposing a backoff time between attempts, limiting the number of failed
authentication attempts, or limiting the total number of attempts.
```

Reasoning: we go talk about failed attempts below, but we never actually said why failed attempts matter.